bot/exts/advent_of_code/_cog.py

```diff
@@ -183,22 +190,25 @@ async def about_aoc(self, ctx: commands.Context) -> None:
         """Respond with an explanation of all things Advent of Code."""
         await ctx.send(embed=self.cached_about_aoc)
 
-    @commands.guild_only()
-    @adventofcode_group.command(name="join", aliases=("j",), brief="Learn how to join the leaderboard (via DM)")
+    @aoc_slash_group.command(name="join")
```

Comment:
We should add a description parameter here to give a better description for end users. Something like: "Get the join code for our community Advent of Code leaderboard"

Reply:
Donem